src/legacy/ts-jest-transformer.ts

```
@@ -231,15 +231,17 @@ export class TsJestTransformer implements SyncTransformer<TsJestTransformerOptio
       }
     } else if (isJsFile || isTsFile) {
       if (isJsFile && isNodeModule(sourcePath)) {
+        const useESM = transformOptions.supportsStaticESM && transformOptions.transformerConfig.useESM
+        // .mjs extension causes ts.transpileModule to ignore `module: CommonJS`; use .js filename to prevent it
+        const transpileFileName = !useESM
+          ? sourcePath.replace(/\.mjs$/, '.js')
+          : sourcePath
```

ESLint

[error] Replace `⏎··········?·sourcePath.replace(/\.mjs$/,·'.js')⏎·········` with `·?·sourcePath.replace(/\.mjs$/,·'.js')`

(prettier/prettier)